wolfssl/wolfcrypt/settings.h

```diff
@@ -2503,7 +2495,7 @@ extern void uITRON4_free(void *p) ;
 #endif
 
 /* warning for not using harden build options (default with ./configure) */
-#ifndef WC_NO_HARDEN
+#if !defined(WC_NO_HARDEN) && !defined(NO_BIG_INT)
```

Comment:
This too is curious. How is `!defined(NO_BIG_INT)` related to the `!defined(WC_NO_HARDEN)` here?